pxr/imaging/hgiMetal/hgi.mm

```diff
@@ -60,6 +60,7 @@
 , _workToFlush(false)
 {
     if (!_device) {
+#if defined(ARCH_OS_OSX)
         if( TfGetenvBool("HGIMETAL_USE_INTEGRATED_GPU", false)) {
```

Comment:
I'm curious which macs need this override?

Reply:
This is an override for AMD based Macs that also have an Intel graphics for power saving - this has been helpful for testing.